### PR TITLE
Grammar

### PR DESCRIPTION
**Pull request by jsoref**
btw, the repo description:
> TCP port scanner, spews SYN packets asynchronously, scanning entire Internet in under 5 minutes.

should be:
> TCP port scanner, spews SYN packets asynchronously, scanning the entire Internet in under 5 minutes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -375,7 +375,7 @@ or exclude a lot of sub-ranges. This chops up the desired range into hundreds
 of smaller ranges.
 
 This leads to one of the slowest parts of the code. We transmit 10 million
-packets per second, and have to convert an index variable to an IP address
```
Comment (erikeckhardt) [inline README.md:378]: This change is a little dubious, imo. It seems okay with the comma.

File: README.md [hunk 2]
```diff
@@ -415,7 +415,7 @@ heck out of target networks that aren't built for this level of speed. We
 need to spread our traffic evenly over the target.
 
 The way we randomize is simply by encrypting the index variable. By definition,
-encryption is random, and creates a 1-to-1 mapping between the original index
```
Comment (erikeckhardt) [inline README.md:418]: Again, the comma seems okay.

File: README.md [hunk 3]
```diff
@@ -494,7 +494,7 @@ interfering with the transmit thread.
 ## Portability
 
 The code runs well on Linux, Windows, and Mac OS X. All the important bits are
-in standard C (C90). It therefore compiles on Visual Studio with Microsoft's
+in standard C (C90). It, therefore, compiles on Visual Studio with Microsoft's
```
Comment (erikeckhardt) [inline README.md:497]: This change is a bit pedantic. The commas aren't needed, and make it more awkward to read.

File: README.md [hunk 4]
```diff
@@ -182,8 +182,8 @@ with the following command:
 
 Windows doesn't respond with RST packets, so neither of these techniques
 are necessary. However, masscan is still designed to work best using its
-own IP address, so you should run that way when possible, even when its
-not strictly necessary.
+own IP address, so you should run that way when possible, even when it
+isn't strictly necessary.
```
Comment (erikeckhardt) [inline README.md:186]: To respect the original author's phrasing, consider `it's not` instead.

**Comment by robertdavidgraham** [comment]
Because of conflicts, I had to apply these manually. I also rejected a couple ("lookup") or made small changes.

**Comment by jsoref** [comment]
@robertdavidgraham: thanks for the update. Can you update the repo description as noted in https://github.com/robertdavidgraham/masscan/pull/458#issue-336945941? (I don't know of a way to make a PR to change that since it isn't part of the git history.)

**Comment by jsoref** [comment]
@erikeckhardt: sorry... I tend to split things into distinct commits when I expect people to object to individual pieces. In theory it makes them easier to selectively apply. I generally assume that if projects care they'll squash PRs. (I'm not attached to credit. My goal is that the next time I visit a repository, it'll be shinier than the previous time I visited it.)

**Pull request closed without merging**